src/main/resources/hudson/security/GlobalMatrixAuthorizationStrategy/config.jelly

```diff
@@ -139,13 +139,12 @@ THE SOFTWARE.
      <f:helpArea />
     </table>
     <script>
-      (function() {
-        <!-- place master outside the DOM tree so that it won't creep into the submitted form -->
```

Comment:
This is not actually necessary. An extra row in the submission doesn't really hurt, and since principals with no permissions assigned are actually ignored server-side, not doing this will not change the result.